Packs/RedCanary/Integrations/RedCanary/RedCanary.py

```diff
@@ -320,7 +320,7 @@ def get_unacknowledged_detections(t, per_page=50):
             attributes = detection.get('attributes', {})
             # If 'last_acknowledged_at' and 'last_acknowledged_by' are in attributes,
             # the detection is acknowledged and should not create a new incident.
-            if attributes.get('last_acknowledged_at') and attributes.get('last_acknowledged_by'):
+            if not (attributes.get('last_acknowledged_at') and attributes.get('last_acknowledged_by')):
```

Comment:
if I remember correctly, I think that in some cases only one of them was filled out so better to check that both not defined.